.eslintignore

```diff
@@ -19,12 +19,6 @@
 /packages/mui-icons-material/src/*.js
```

Comment:
`tsup.config` should also be deleted

Reply:
`scripts/pigment-license` can be deleted.

Reply:
I'll update with latest master and remove these, thanks!

Reply:
I also remove the tsup dependency